dev_guide/pod_autoscaling.adoc

```diff
@@ -65,10 +65,9 @@ and scales up or down accordingly. The scaling will occur at a regular interval,
 but it may take one to two minutes before metrics make their way into Heapster.
 
 For replication controllers, this scaling corresponds directly to the replicas
-of the replication controller. For deployment configurations, this will scale
-the latest deployment, or, if no deployment is present, the replication
-controller template in the deployment configuration. Note that scaling will
-occur if the latest deployment is marked as `Complete`.
+of the replication controller. For deployment configurations, scaling corresponds
+directly to the replica count of the deployment configuration. Note that auto-
+scaling applies only to the latest deployment in the `Complete` phase.
```

Comment:
@stevekuznetsov In the GH preview at least, this is showing up as "auto- scaling" (with a space). Can you check on that?